### DISCUSSION TITLE
Allow more prefixes for Autolink references

### DISCUSSION THREAD

**Discussion by BickelLukas** [community/community#4725] [Other Feature Feedback, Questions, & Ideas] [answered]
I need to create a Autolink reference with the prefix `cons.` (e.g. `cons.7259`)

I get the following error message when trying to save: `Reference prefix must only contain letters, numbers, or -:/#`

Please allow `.` in the reference prefix

**Comment by michellemerrill** [comment, marked as answer]
@BickelLukas Thanks for the feedback! We're looking into this and will post here with updates!

**Reply by larsxschneider** [reply]
@BickelLukas we recently addressed the issue: https://github.blog/changelog/2021-11-04-more-flexible-autolinks/

Does this fix the problem for you?

**Reply by BickelLukas** [reply]
@larsxschneider Hi, yes this fixes the Error. Thanks!
